stance_detection.py

```diff
@@ -36,14 +37,15 @@ def __init__(self):
 
     def gen_training_features(self, bodies_fpath, stances_fpath):
         self._read(bodies_fpath, stances_fpath)
-        unigrams = self._train_ngrams(1)
+        self._ngrams = self._train_ngrams(1)
```

Comment:
I had it named unigrams, because we pass in 1 to _train_ngrams, so it will return unigrams. I thought we wanted to do bigrams and unigrams, so I was thinking we'd have self._unigrams, and self._bigrams. which would call _train_ngrams(1) and _train_ngrams(2) respectively.